### PR TITLE
page: Bring back the product page

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
Recently removed as it was a copy of the pricing page, this change
brings the product page back and starts the first iteration on the
content for the page.

### CODE REVIEW

File: src/product.njk [hunk 1]
```diff
@@ -56,29 +56,38 @@ heroLg: true
...
-
-<div class="about w-full ff-bg-light pb-32">
-    <div class="container m-auto text-center max-w-4xl">
-        <!-- Cloud Feature List -->
-        <div id="tab-cloud" class="ff-tab-content ff--active">{{ cloudFeaturesHtml | safe }}</div>
+		With centralized user management, and role based access control RBAC and centrilized users</p>
+	</div>
+    	<div class="text-left">
+		<h1>Team-Based Collaboration</h1>
+		<p></p>
```
Comment (ZJvandeWeg) [inline src/product.njk:78]: @sammachin @knolleary Would appreciate input on what to put in these paragraphs. A first iteration is fine, but want to have a first iteration merged in the next hour or two.

File: src/product.njk [hunk 2]
```diff
@@ -56,29 +56,38 @@ heroLg: true
...
-{% endset -%}
+<div class="w-full ff-bg-light pb-32">
+    <div class="m-auto max-w-4xl">
+    	<div class="text-left">
+    		<h1>Sub-Minute setup</h1>
+	    	<p>With FlowForge deploying a new Node-RED project takes less than a minute. No configuration of proxies and setting up DNS, get going in less time it takes to make a cup of tee.</p>
+    	</div>
+    	<div class="text-right">
+		<h1>Entreprise-Ready</h1>
+		<p>Stacks, templates, etc
```
Comment (ZJvandeWeg) [inline src/product.njk:72]: Manage the versions of Node-RED being deployed and their resources. Customise the palette of nodes that are preinstalled, provide a set of default flows, and change the look and feel of the editor.

File: src/product.njk [hunk 3]
```diff
@@ -56,29 +56,38 @@ heroLg: true
...
+	<div class="text-right">
+		<h1>Node Catalog</h1>
+		<p>Leverage over 3000 custom nodes already developed to connect the physical world to digital servives and back again.</p>
+	</div>
+    	<div class="text-left">
+		<h1>Team-Based Collaboration</h1>
+		<p>Centralized user management to collaborate with multiple users</p>
+	</div>
+	<div class="text-right">
+		<h1>Audit Log</h1>
```
Comment (ZJvandeWeg) [inline src/product.njk:89]: @sammachin There's no documentation on this feature, can you add this?

File: src/product.njk [hunk 4]
```diff
@@ -56,29 +56,38 @@ heroLg: true
...
-</div>
-{% endset -%}
+<div class="w-full ff-bg-light pb-32">
+    <div class="m-auto max-w-4xl">
+    	<div class="text-left">
+    		<h1>Sub-Minute setup</h1>
+	    	<p>With FlowForge deploying a new Node-RED project takes less than a minute. No configuration of proxies and setting up DNS, get going in less time it takes to make a cup of tee.</p>
+    	</div>
+    	<div class="text-right">
+		<h1>Entreprise-Ready</h1>
```
Comment (sammachin) [inline src/product.njk:71]:
```suggestion
		<h1>Enterprise-Ready</h1>
```

File: src/product.njk [hunk 5]
```diff
@@ -37,15 +37,15 @@ heroLg: true
             </div>
             <div class="company-value">
                 <img src="../images/pictograms/security.png" />
-                <h5>SSO</h5>
+                <h5>Entreprise Ready</h5>
```
Comment (knolleary) [inline src/product.njk:40]:
```suggestion
                <h5>Enterprise Ready</h5>
```

File: src/_includes/layouts/base.njk [hunk 6]
```diff
@@ -42,7 +42,7 @@
 
                     <ul id="nav-content" class="hidden absolute top-8 right-0 border shadow-sm sm:inline sm:shadow-none w-36 z-10 sm:border-0 bg-white sm:bg-transparent sm:w-auto rounded sm:static sm:float-none sm:flex sm:flex-row flex-col items-center justify-between font-medium text no-underline">
                         <li class="flex"><a class="flex-grow sm:px-4 sm:pl-0 sm:py-2" href="/about">about</a></li>
-                        {# <li class="flex"><a href="/product">platform</a></li> #}
+                        <li class="flex"><a href="/product">product</a></li>
                         <li class="flex"><a href="/pricing">pricing</a></li>
```
Comment (knolleary) [inline src/_includes/layouts/base.njk:46]:
```suggestion
                        <li class="flex"><a class="flex-grow sm:px-4 sm:pl-0 sm:py-2" href="/product">product</a></li>
                        <li class="flex"><a href="/pricing">pricing</a></li>
```

File: src/_includes/layouts/base.njk [hunk 7]
```diff
@@ -42,7 +42,7 @@
 
                     <ul id="nav-content" class="hidden absolute top-8 right-0 border shadow-sm sm:inline sm:shadow-none w-36 z-10 sm:border-0 bg-white sm:bg-transparent sm:w-auto rounded sm:static sm:float-none sm:flex sm:flex-row flex-col items-center justify-between font-medium text no-underline">
                         <li class="flex"><a class="flex-grow sm:px-4 sm:pl-0 sm:py-2" href="/about">about</a></li>
```
Comment (knolleary) [inline src/_includes/layouts/base.njk:44]:
```suggestion
```

File: src/product.njk (hunk 3 above)
Comment (sammachin) [reply src/product.njk:83]: yes I'll add an inital section now, will probably need to evolve it as we gain more events in the audit log

**Review by knolleary — APPROVED** [review]

File: src/product.njk (hunk 3 above)
Comment (hardillb) [reply src/product.njk]: Joe was starting a list of Audit Events (https://docs.google.com/spreadsheets/d/1e1tHRGI01J65iTetjWcJj2sMVXYVm7kaIg4y_VnGQno/edit#gid=0)

**Pull request merged**